app/views/spotlight/exhibits/_exhibit_card.html.erb

```diff
@@ -1,7 +1,11 @@
 <div class="col-xs-12 col-sm-4 exhibit-card">
   <div class="flipper">
     <div class="card-front card-face">
-      <%= image_tag(exhibit.thumbnail.image.square.url) if exhibit.thumbnail.present? %>
+      <% if exhibit.thumbnail.present? %>
+        <%= image_tag(exhibit.thumbnail.image.square.url) %>
+      <% else %>
+        <%= image_tag 'spotlight/default_thumbnail.jpg' %>
```

Comment:
Would it be helpful to make this configurable so that people can use their own image, or is it possible to just override the image itself? (I'm not 100% sure how that all works w/ engines and the asset pipeline)